lesson_06/homework_06.py

```diff
@@ -51,22 +103,45 @@
 У кожній спробі гравець вводить своє припущення, після чого програма повідомляє, чи
 було припущення занадто великим або занадто малим, чи гравець вгадав число.
 """
-import random
 secret_number = random.randint(1, 20)
+print(secret_number)
 guesses = 0
 max_guesses = 5
-print("Вгадайте число від 1 до 20 за 5 спроб!")
+
+while True:
+    guess = int(input("Вгадайте число від 1 до 20: "))
+    if secret_number != guess:
+        guesses += 1
+        if guess >= secret_number:
+            print("Занадто велике число!")
+        elif guess <= secret_number:
+            print("Занадто маленьке число!")
+
+        if guesses == max_guesses:
+            print(f"Іншого разу пощастить! Число було: {secret_number}.")
+            break
+        print(f"У Вас залишилось {max_guesses - guesses} спроб! ")
+    else:
+        print("Вітаємо! Ви вгадали число!")
+        break
+
 
 # task 9
 """  Задача з використанням циклу for та continue. Задано список фруктів 'fruits'
 потрібно вивести на екран всі елементи списку, окрім "orange".
 """
 fruits = ["apple", "banana", "orange", "grape", "mango"]
 
+for item in fruits:
+    if item != 'orange':
+        print(item)
+        continue
```

Comment:
У вашому випадку continue не треба взагалі
```suggestion
for item in fruits:
    if item != 'orange':
        print(item)
```
але малося на увазі
```suggestion
for item in fruits:
    if item == 'orange':
        continue
   print(item)
```

Reply:
я теж подумала, що взагалі не потрібен continue тому що воно і так працювало) дякую за коментар